openshift/templates/deploy.dc.yaml

```diff
@@ -77,12 +81,12 @@ objects:
             - image: ${DOCKER_REGISTRY}/${PROJECT_TOOLS}/${NAME}:${SUFFIX}
               name: ${NAME}
               env:
-                - name: REACT_APP_API_BASE_URL
+                - name: env.api-base-url
```

Comment:
We still need to keep the name as REACT_APP_API_BASE_URL, REACT_APP_FIDER_LINK. So no need to change anything here

Reply:
Good catches!  :)